docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -2075,6 +2075,108 @@ Converts a string to its lowercase equivalent.
 --------------------------------------------------
 // NOTCONSOLE
 
+[[pipeline-processor]]
+=== Pipeline Processor
+Executes another pipeline.
+
+[[pipeline-options]]
+.Pipeline Options
+[options="header"]
+|======
+| Name             | Required  | Default  | Description
+| `pipeline`       | yes       | -        | The name of the pipeline to execute
+|======
+
```

Comment:
Can I suggest to use `pipelineA` and `pipelineB` (or the like) as examples to avoid inner/outer which (to me at least) suggests a required relationship between the two pipelines.